import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,54 @@
+import { formatJSONResponse } from '@libs/api-gateway';
+import { headers } from '@libs/headers';
+import { middyfy } from '@libs/lambda';
+
+import * as AWS from 'aws-sdk';
+
+export const importProductsFile = async(event: any) => {
+  const { name } = event.queryStringParameters;
+
+  if (!name) {
+    console.log('Error: no filename provided');
+    return formatJSONResponse({
+      response: 'Error: no filename provided',
+      statusCode: 500,
+      headers,
+    });
+  }
+
+  const s3 = new AWS.S3({ region: 'eu-central-1' });
+
+  try {
+    const BUCKET_NAME = process.env.BUCKET_NAME;
+    const url = await s3.getSignedUrlPromise('putObject', {
```

Comment:
As it was mentioned during one of the previous lectures there is already AWS SDK _v3_ available, you could try it in the next tasks

Here is the link which might be useful, it describes the key differences between _v2_ and _v3_: https://serverless.pub/migrating-to-aws-sdk-v3/